pkg/machinery/config/kernel.go

```diff
@@ -0,0 +1,25 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+package config
```

Comment:
feels like this probably doesn't belong to `config` package, should it be better `package kernel` in `machinery` ? with `kernel.DefaultArgs` ?

Reply:
Yep, if you think it deserves it's own personal folder :)